src/main/java/org/alliancegenome/curation_api/services/validation/dto/associations/EvidenceAssociationDTOValidator.java

```diff
@@ -22,7 +22,8 @@ public class EvidenceAssociationDTOValidator extends BaseDTOValidator {
 
 	public <E extends EvidenceAssociation, D extends EvidenceAssociationDTO> ObjectResponse<E> validateEvidenceAssociationDTO(E association, D dto) {
 		ObjectResponse<E> assocResponse = validateAuditedObjectDTO(association, dto);
-
+		association = assocResponse.getEntity();
```

Comment:
The association is directly changed in the validateAuditedObjectDTO, so need to pull it off the ObjectResponse.

Reply:
Yep, sorry - I misread the change and thought this was being removed instead of added

Reply:
Oh hang on, this is my PR. So yes, this needed to be added for that reason.